projects/05/Computer.hdl

```diff
@@ -19,5 +19,8 @@ CHIP Computer {
     IN reset;
 
     PARTS:
-    // Put your code here:
+    CPU(inM=inMemory, instruction=nextInstruction, reset=reset,
+        outM=MValue, writeM=shouldWriteM, addressM=addressOfM, pc=nextAddress);
+    ROM32K(address=nextAddress, out=nextInstruction);
+    Memory(in=MValue, load=shouldWriteM, address=addressOfM, out=inMemory);

```

Comment:
This is all correct! 🎉 

The CPU is working fine and this is the correct way to hook it up to the ROM and RAM. The problem is that the screen isn’t storing data when `load` is true, so even though all the right data is being sent to to the screen during `ComputerRect.tst`, it isn’t being remembered. See my next comment.